main/src/main/scala/sbt/internal/server/BuildServerReporter.scala

```diff
@@ -82,7 +101,7 @@ final class BuildServerReporterImpl(
   import sbt.internal.inc.JavaInterfaceUtil._
 
   private lazy val exchange = StandardMain.exchange
-  private val problemsByFile = mutable.Map[Path, Vector[Diagnostic]]()
+  private val problemsByFile = mutable.Map[VirtualFileRef, Vector[Problem]]()
```

Comment:
Add a comment to explain which files? The source of the generator or the generated files?

Reply:
Maybe rename `problemsBySourceFiles`